Mage.Client/src/main/java/mage/client/dialog/NewTournamentDialog.java

```diff
@@ -99,7 +96,7 @@ public void showDialog(UUID roomId) {
                 tournamentPlayerPanel.init(i++);
             }
             cbAllowSpectators.setSelected(true);
-            cbPlaneChase.setSelected(false);
+            //cbPlaneChase.setSelected(false);
```

Comment:
There are more places with commented out code that weren't fixed by your latest commit